va/va.h

```diff
@@ -1401,6 +1408,21 @@ typedef union _VAConfigAttribValContextPriority {
     uint32_t value;
 } VAConfigAttribValContextPriority;
 
+/** brief Attribute value VAConfigAttribEncPerMBControl */
+typedef union VAConfigAttribValEncPerMBControl {
+    struct {
+        /** \brief whether to support dela qp per block */
+        uint32_t delta_qp_support         : 1;
+        /** \brief supported size of delta qp block */
+        uint32_t log2_delta_qp_block_size : 4;
+        /** \brief size of delta qp per block in bytes*/
+        uint32_t delta_qp_per_block_size  : 3;
+        /** \brief reserved bit for future, must be zero */
+        uint32_t reserved                 : 24;
+        } bits;
+        uint32_t value;
+} VAConfigAttribValEncPerMBControl;
```

Comment:
It is 'PerMB' in the structure name, however it is 'per block' in the comment for each member. Is this structure added for ' mb' or 'block' ?

Reply:
Change it to block